unitTests/ui/ComponentEventsTest.ts

```diff
@@ -28,6 +29,36 @@ export function ComponentEventsTest() {
       expect(spy).toHaveBeenCalledWith({ bar: 'baz' });
     });
 
+    it('should execute handler with the originalEvent if its wrapped by jQuery and does not have any param', function() {
+      // Setup.
+      window['Coveo']['$'] = $;
+
+      test.cmp.enable();
+      test.cmp.bind.on(test.env.root, 'click', spy);
+
+      // Trigger native click event, expecting it to be wrapped by jQuery.
+      test.env.root.click();
+      // It should call the spy with the native Event for first and only param.
+      expect(spy).toHaveBeenCalled();
+      expect(spy.calls.argsFor(0)[0] instanceof MouseEvent).toBeTruthy();
+      expect((spy.calls.argsFor(0)[0] as MouseEvent).type).toBe('click');
+
+      // Trigger jQuery click event, without any params.
+      $(test.env.root).click();
+      // It should call the spy without any params.
+      expect(spy).toHaveBeenCalledTimes(2);
+      expect(spy.calls.argsFor(1).length).toBe(0);
+
+      // Trigger jQuery click event, with some param.
+      $$(test.env.root).trigger('click', { bar: 'baz' });
```

Comment:
Why do we use `$$` here but just `$` on line 47, even though both are meant to trigger JQuery events? Should they not use the same mechanism?